content/en/docs/reference/scheduling/config.md

```diff
@@ -35,9 +34,10 @@ clientConnection:
 
   {{< note >}}
   KubeSchedulerConfiguration [v1beta2](/docs/reference/config-api/kube-scheduler-config.v1beta2/)
-  is deprecated in v1.25 and will be removed in v1.26. Please migrate KubeSchedulerConfiguration to
-  [v1beta3](/docs/reference/config-api/kube-scheduler-config.v1beta3/) or [v1](/docs/reference/config-api/kube-scheduler-config.v1/)
-  before upgrading Kubernetes to v1.25.
+  is deprecated in v1.25 and will be removed in v1.28. 
+  KubeSchedulerConfiguration [v1beta3](/docs/reference/config-api/kube-scheduler-config.v1beta3/)
+  is deprecated in v1.26 and will be removed in v1.29.
+  Please migrate KubeSchedulerConfiguration to [v1](/docs/reference/config-api/kube-scheduler-config.v1/).
```

Comment:
Is this true, and would SIG Architecture confirm this as a firm commitment by the Kubernetes project?

Reply:
We're letting users know about the deprecation on:
- deprecation warning logs: https://github.com/kubernetes/kubernetes/blob/1def77f05cf701439f027fb24b229d9696971323/cmd/kube-scheduler/app/options/configfile.go#L56-L59
- PR with the release note (v1.26): https://github.com/kubernetes/kubernetes/pull/112257

As following the deprecation rule, we were waiting for 3 releases; and v1.28 (v1beta2) and v1.29 are time to remove them completely.
https://kubernetes.io/docs/reference/using-api/deprecation-policy/#component-config-structures

Reply:
So, the current website is based on pretty old information, we've already done the deprecation actually, and this PR is just reflecting them on the website